tests/rptest/util.py

```diff
@@ -488,3 +489,49 @@ def ssh_output_stderr(source_service,
     source_service.logger.debug(
         f"Returning ssh command output:\n{stdoutdata}\n")
     return stdoutdata, stderrdata
+
+
+def bg_thread_cm(func) -> Callable[..., ContextManager]:
```

Comment:
wow :)

Reply:
If you wow is about type annotations, without them linter goes wild.
Not quite sure why, it's okay with similar things in contexlib.

Reply:
the wow is about this pretty cool useful refactor (nothing to do with type annotations).